pkgs/development/python-modules/pylnk3/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, twine
+, invoke
+}:
+
+buildPythonPackage rec {
+  pname = "pylnk3";
+  version = "0.4.2";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "pylnk3";
+    sha256 = "sha256-yu4BNvYai3iBVNyOfAOsLd5XrcFw8cR4arRyFJHKbpk=";
+  };
+
+   propagatedBuildInputs = [
+     pytest
+     twine
+     invoke
+   ];
+
+  pythonImportsCheck = [
+    "pylnk3"
+  ];
+
+  meta = with lib; {
+    description = "Python library for reading and writing Windows shortcut files (.lnk)";
+    homepage = "https://github.com/strayge/pylnk";
+    license = with licenses; [ lgpl3 ];
```

I can't find any mentioning that later releases of the LGPL are supported.

```suggestion
    license = with licenses; [ lgpl3Only ];
```